Author: Pino Toscano <toscano.pino@tiscali.it>
Description: Replace sys/mount.h include with sys/statfs.h on GNU/kFreeBSD.
 Due to Debian #704598, GNU/kFreeBSD' sys/mount.h cannot be used in
 C90 C sources, such as plat_freebsd.c when compiled with default CFLAGS
 provided with kdelibs.
 When using GNU libc, use sys/statfs.h instead of sys/mount.h; the former
 should provide fstatfs.
Forwarded: no
Last-Update: 2014-04-09

--- a/wmlib/plat_freebsd.c
+++ b/wmlib/plat_freebsd.c
@@ -36,7 +36,11 @@
 #include <unistd.h>
 #include <sys/types.h>
 #include <sys/param.h>
-#include <sys/mount.h>
+#ifdef __GLIBC__
+# include <sys/statfs.h>
+#else
+# include <sys/mount.h>
+#endif
 #include <sys/stat.h>
 
 #include "include/wm_config.h"
